src/rsz/test/rebuffer1.tcl

```diff
@@ -0,0 +1,41 @@
+# rebuffering test in flattened flow
+source "helpers.tcl"
+
+define_corners fast slow
+read_liberty -corner slow Nangate45/Nangate45_slow.lib
+read_liberty -corner fast Nangate45/Nangate45_fast.lib
+read_lef Nangate45/Nangate45.lef
+read_verilog rebuffer1.v
+link_design top
+
+#place the design
+initialize_floorplan -die_area "0 0 40 1200" -core_area "0 0 40 1200" \
+  -site FreePDK45_38x28_10R_NP_162NW_34O
+global_placement -skip_nesterov_place
```

Comment:
I think we should use a preplaced .def so that the test isn't sensitive to GPL changes